### ISSUE TITLE
Evaluate validation set

### ISSUE THREAD

**Issue by epwalsh** [epwalsh/pytorch-crf#14]
- [x] create eval module
- [x] create unit tests for eval module
- [x] implement evaluation block in train.py
- [x] add macro average precision, recall, f1, and item, instance accuracy
